extension/docs/uddsketch.md

```diff
@@ -111,7 +112,59 @@ FROM daily_rain;
 (4 rows)
 ```
 
+## Example Using TimeScale Continuous Aggregates (uddsketch-cagg-example)
+To have a UddSketch over a PostgresQL table which automatically updates as more data is added, we can make use of continuous aggregates.  First, let us create a simple hypertable:
+
+```SQL ,non-transactional,ignore-output
+SET TIME ZONE 'UTC';
+CREATE TABLE test(time TIMESTAMPTZ, value DOUBLE PRECISION);
+SELECT create_hypertable('test', 'time');
+```
+
+Now we'll create a continuous aggregate which will group all the points for each week into a UddSketch:
+```SQL ,non-transactional,ignore-output
+CREATE MATERIALIZED VIEW weekly_sketch
+WITH (timescaledb.continuous)
+AS SELECT
+    time_bucket('1 week'::interval, time) as week,
```

Comment:
I think time_bucket does weird stuff with weeks, could we do a couple days or a single day or something like that instead? (Just so we don't end up with people doing things that get odd results and complain)